.gitignore

```
@@ -34,7 +34,7 @@ robots.txt
 /nbproject/private/
 
 #Ignore config files from VSCode
-/.vscode/
+#/.vscode/
```

Please add explicitly the `.vscode/launch.json` exception to the exclusion rule above instead of allowing all the files.
```suggestion
/.vscode/
!/.vscode/launch.json
```
I think this is the right syntax, please verify.

---

It is.